contracts/0.8.25/vaults/Dashboard.sol

```diff
@@ -75,15 +75,17 @@ contract Dashboard is Permissions {
     /**
      * @notice Constructor sets the stETH, WETH, and WSTETH token addresses.
      * @param _wETH Address of the weth token contract.
+     * @param _wstETH Address of the wsteth token contract.
      * @param _lidoLocator Address of the Lido locator contract.
      */
-    constructor(address _wETH, address _lidoLocator) {
+    constructor(address _wETH, address _wstETH, address _lidoLocator) {
```

Comment:
wstETH we do have in locator now

Reply:
it is in the new locator only which is to be set only upon Aragon voting execution